example/ios/Podfile.lock

```diff
@@ -0,0 +1,29 @@
+PODS:
+  - Flutter (1.0.0)
+  - truecaller_sdk (0.0.1):
```

Comment:
what does this 0.0.1 stand for? coz our latest version is 0.1.0, so I hope this won't conflict.

Reply:
this was more or less auto downloaded, I'll check again